lib/WP_Auth0_LoginManager.php

```diff
@@ -170,15 +170,15 @@ public function init_auth0() {
       }
     } catch (WP_Auth0_LoginFlowValidationException $e) {
 
-      $msg = __( 'There was a problem with your log in', 'wp-auth0' );
+      $msg = __( 'There was a problem with your log in. ', 'wp-auth0' );
```

Comment:
i18n?

Reply:
@lbalmaceda - yes?

Reply:
Should probably change these back as this will break existing i18n for this message as it's changed.